### ISSUE TITLE
Why validation loss function varies much compare to training loss function

### ISSUE THREAD

**Issue by sneh-debug** [ellisdg/3DUnetCNN#261]
@ellisdg why validation loss fluctuations much compare to validation loss? I have used 219 images for training. Is the model overfitting?

**Comment by ellisdg** [comment]
This is probably due to the smaller size of the validation set compared to the training set. If you increase your validation set size, it should be more stable. Sorry for the late response.
